### DISCUSSION TITLE
Not automatically starting up on boot after 1.20

### DISCUSSION THREAD

**Discussion by 1c383r9** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#874] [General]
I reinstalled the helper and it still doesn't autostart. I checked the activity monitor and found no aldente process.

Any help?

**Comment by MatthiasKerbl** [comment]
Hi @2ndMessiah,

We are currently investigating this issue and will try to fix it as soon as possible.

Cheers,
Matthias

**Reply by 1c383r9** [reply]
Known issue??

Looking forward to the solution.

Appreciate it.

**Comment by 1c383r9** [comment]
I noticed that the "autostart" switch in the settings was turned off after the 1.20 update... 

Haven't tested if this switch really works, but it seems to be the case.

**Reply by MatthiasKerbl** [reply]
That's odd. Let me know if Launch at login works for you.

**Reply by 1c383r9** [reply]
Tested. The switch works. The odd point is that the autostart switch was turned off after the upgrade. I don't think I manually did this.
